services/distribution/src/main/resources/application.yaml

```diff
@@ -115,7 +126,8 @@ services:
         max-exposure-detections-per-interval: ${ANDROID_EXPOSURE_DETECTION_MAX_ED_PER_INTERVAL:6}
         overall-timeout-in-seconds: ${ANDROID_EXPOSURE_DETECTION_OVERALL_TIMEOUT_IN_SECONDS:900}
 
-
+remote-file-client:
+  base-url: 'https://obs.eu-de.otc.t-systems.com/obs-cwa-public-dev'
```

Comment:
Is there an impact to using this URL when the app is deployed in differnet environments (INT, DEV, WRU ) ?

Reply:
I removed this since it was used for local testing, and replaced with a different approach